marklogic-data-hub-central/ui/src/util/data-conversion.tsx

```diff
@@ -1,4 +1,8 @@
 import { xmlParser } from './../util/xml-parser';
+import { Tooltip } from 'antd';
+import React, { useState, useContext } from 'react';
```

Comment:
data-conversion file is meant to be a file where payloads are parsed. If you have JSX or React code, it's better to keep it within the component for maintaining, testing and debugging

Reply:
Fixed. Thanks!